src/__tests__/search/search.types.spec.ts

```diff
@@ -7,11 +7,13 @@ import {
 	SearchHitMeta as ProtoSearchHitMeta,
 	SearchMetadata as ProtoSearchMetadata,
 	SearchResponse as ProtoSearchResponse,
+	Match as ProtoMatch,
+	MatchField as ProtoMatchField,
 } from "../../proto/server/v1/api_pb";
 import { TestTigrisService } from "../test-service";
 import { IBook } from "../tigris.rpc.spec";
 import * as google_protobuf_timestamp_pb from "google-protobuf/google/protobuf/timestamp_pb";
-import { SearchResult } from "../../search/result";
+import { TextMatchInfo, SearchResult, DocMeta } from "../../search";
```

Comment:
Is this updated SearchMeta?

Reply:
Yes.

```
export class DocMeta {
	private readonly _createdAt: Date;
	private readonly _updatedAt: Date;
	private readonly _textMatch: TextMatchInfo;
}
```

Reply:
Added it [here](https://github.com/tigrisdata/tigris-client-ts/pull/263/commits/ec23573579e268583134d6a08f1d35813b3f32dc)